packages/twenty-server/src/modules/attachment/standard-objects/attachment.object-metadata.ts

```diff
@@ -55,6 +57,36 @@ export class AttachmentObjectMetadata extends BaseObjectMetadata {
   })
   type: string;
 
+  @FieldMetadata({
+    standardId: attachmentStandardFieldIds.storageDriverType,
+    type: FieldMetadataType.SELECT,
+    label: 'Storage driver type',
+    description: 'Attachment storage driver type',
+    icon: 'IconCloud',
+    options: [
+      {
+        value: StorageDriverType.Local,
+        label: 'Local',
+        position: 0,
+        color: 'blue',
+      },
+      {
+        value: StorageDriverType.S3,
+        label: 'S3',
+        position: 1,
+        color: 'yellow',
+      },
+      {
+        value: StorageDriverType.Gmail,
+        label: 'Gmail',
+        position: 2,
+        color: 'red',
+      },
+    ],
+  })
+  @IsNullable()
+  storageDriverType: StorageDriverType;
```

Comment:
Is it ok to reuse the existing enum here?

Reply:
It is okay